src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -197,12 +198,19 @@ export class NetCoreDebugHelper implements DebugHelper {
     }
 
     private async acquireDebuggers(platformOS: PlatformOS): Promise<void> {
-        if (platformOS === 'Windows') {
-            await installDebuggerIfNecessary('win7-x64', 'latest');
-        } else {
-            await installDebuggerIfNecessary('linux-x64', 'latest');
-            await installDebuggerIfNecessary('linux-musl-x64', 'latest');
-        }
+        await window.withProgress(
+            {
+                location: ProgressLocation.Notification,
+                title: localize('vscode-docker.debug.netcore.acquiringDebuggers', 'Acquiring VSDBG...'),

```

Comment:
"Installing debugger into the container..." maybe?

Not sure if everybody knows what "VSDBG" is...

Reply:
In this it's installing locally, but I agree about the name change. I'll do "Acquiring .NET debugger..."